contents/javascript/Javascript-framework-VueJs.md

```diff
@@ -172,7 +172,7 @@ In this case, only the `root` component can be accessed in Vue while the rest ar
 
     /* Inside todo-list component */
     template: {
-      on:increased-count="updateCount"
+      <item v-on:increased-count="updateCount" v-for="item in items"/>
```

Comment:
`template: '...' `, just noticed the one on top uses `{ ... }` as well, could change that too

Reply:
I standardise to use backticks instead of single quotes, since the top one has multiple lines

Reply:
LGTM 👍